app/controllers/users/confirmations_controller.rb

```diff
@@ -8,6 +8,7 @@ def show
     if resource.errors.empty?
       notify_school_primary_contact
       sign_in(resource)
+      @current_user = current_user
```

Comment:
Why do we need this here?

In `ApplicationController` we do
```
before_action :set_current_user, unless: :devise_controller?
```

Do we want to change that?

Reply:
So that we can do [this](https://github.com/DFE-Digital/early-careers-framework/pull/182/files/63078a0de6e6197e916bf41831d5a4f3bca1a8dd#diff-8d1216620140287a4b4580989961295ee24e8381011a36ff54d69e79f7a846c6L6). We're not actually using confirmations of any kind, and I'm not sure we plan to